src/plantPage/PlantCard.js

```diff
@@ -11,35 +11,50 @@ import {
   CardTitle,
   CardHeader,
   CardLine,
+  Container,
 } from '../common/CardComponents'
+import { EditAlt } from 'styled-icons/boxicons-regular/'
+import Notes from './Notes'
 
 const StyledCard = styled(Card)`
-  width: 70%;
+  width: 80%;
 `
 
-const Container = styled(NavLink)``
+const LinkContainer = styled(NavLink)``
 
 export function PlantCard({
-  img,
-  alt,
-  title,
-  id,
-  onBookmark,
-  isBookmarked,
-  tags,
-  getOptionLabel,
+  plant,
   selection,
   matchInfo,
+  onBookmark,
+  getOptionLabel,
+  onUserInput,
 }) {
+  const [editMode, setEditMode] = useState(false)
+
+  const { img, alt, title, id, isBookmarked } = plant
+  const tags = plant.tagList
+
+  function handleEditMode() {
+    setEditMode(!editMode)
+  }
+
   return (
     <StyledCard>
-      <Container to={'/detail/' + id}>
+      <LinkContainer to={'/detail/' + id}>
         <CardPlantImage src={img} alt={alt} />
-      </Container>
+      </LinkContainer>
       <CardTextbox>
         <CardHeader>
           <CardTitle>{title}</CardTitle>
-          <Bookmark onClick={onBookmark} id={id} isBookmarked={isBookmarked} />
+          <Container>
+            <EditAlt size={20} onClick={handleEditMode} />
```

Comment:
Here you can just write the function in the onClick directly, because it is so small. 
`<EditAlt size={20} onClick={() => setEditMode(!editMode)} />`